internal/formats/spdx22json/decoder_test.go

```diff
@@ -61,6 +63,12 @@ func TestSPDXJSONDecoder(t *testing.T) {
 			f, err := os.Open("test-fixtures/spdx/" + test.path)
 			assert.NoError(t, err)
 
+			if !test.fail {
+				err = Format().Validate(f)
+				require.NoError(t, err)
+				_, err = f.Seek(0, io.SeekStart)
+			}
+
```

Comment:
What is the intent of this block?

Reply:
The intent is to use SPDX's lib to validate our home-brew encoder, no changes to our encoder were made but it is good to have fallback tests for the future.

Reply:
I don't understand yet... How does this fit in with the conversion functionality? And what is this guarding us from?

Reply:
This is making sure future changes to our SPDX encoder will follow their (SPDX) official specs.

Reply:
Actually, I think a better place for it is with encode/decode tests.